### PR TITLE
[34355] EE: Trial creation modal changes size when error is shown

### PR DESCRIPTION
**Pull request by bsatarnejad**
Modal size should ideally not be changed while there is an error shown.

https://community.openproject.com/projects/openproject/work_packages/34355/activity

### CODE REVIEW

**Review by ulferts — CHANGES_REQUESTED** [review]
This will change the width of the form--field-instructions on all of the pages, e.g. the change leads to this:

image

being turned into this:

image

Is it possible to have the combination of `max-width` and `width` in the style so that it is not stretched to max-width all the time but cannot exceed the max-width?

**Comment by bsatarnejad** [comment]
Hi @ulferts yes i tried both width and max-width, but the result is different in several places, for example in this image the width is set to min-content and max-width=500px or in other local tests width=110px and max-width=500px. There are not any identical results, i think it depends on its parent width. As we have this problem in this component, i prefer to add a sass file for its specific styles and fix this bug there. What's your opinion?

Screenshot from 2020-10-01 08-27-28

Screenshot from 2020-10-01 08-13-50

**Comment by ulferts** [comment]
Sounds good @bsatarnejad.

File: frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.ts [hunk 1]
```diff
@@ -37,7 +37,8 @@ const newsletterURL = 'https://www.openproject.com/newsletter/';
```
Comment (openprojectci) [inline frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.ts:37]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/ee3402c46a0766828c603a1c6e252ae89c035fac/frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

**Review by ulferts — APPROVED** [review]
:+1:

**Pull request merged**